Toolbox/ArcSDM.SiteReductionTool.pyt.xml

```diff
@@ -1 +1,2 @@
ction}"><dialogReference>&lt;DIV STYLE="text-align:Left;"&gt;&lt;DIV&gt;&lt;P&gt;&lt;SPAN&gt;Select to do random thinning. &lt;/SPAN&gt;&lt;/P&gt;&lt;/DIV&gt;&lt;/DIV&gt;</dialogReference></param><param name="Random_percentage_selection" displayname="Random percentage selection" type="Optional" direction="Input" datatype="Long" expression="{Random_percentage_selection}"><dialogReference>&lt;DIV STYLE="text-align:Left;"&gt;&lt;DIV&gt;&lt;P&gt;&lt;SPAN&gt;Random thinning percentage to keep. &lt;/SPAN&gt;&lt;/P&gt;&lt;/DIV&gt;&lt;/DIV&gt;</dialogReference></param><param name="layerSelection" displayname="Save selection as a new layer" type="Required" direction="Output" datatype="Feature Layer" expression="layerSelection"><dialogReference>&lt;DIV STYLE="text-align:Left;"&gt;&lt;DIV&gt;&lt;DIV&gt;&lt;P&gt;&lt;SPAN&gt;Return thinned &lt;/SPAN&gt;&lt;SPAN&gt;selection as a &lt;/SPAN&gt;&lt;SPAN&gt;new l&lt;/SPAN&gt;&lt;SPAN&gt;ayer&lt;/SPAN&gt;&lt;SPAN&gt;.&lt;/SPAN&gt;&lt;/P&gt;&lt;/DIV&gt;&lt;/DIV&gt;&lt;/DIV&gt;</dialogReference></param></parameters><summary>&lt;DIV STYLE="text-align:Left;"&gt;&lt;DIV&gt;&lt;DIV&gt;&lt;P&gt;&lt;SPAN&gt;Reduce number of training sites by random subset of sites or by thinning to one cell per unit areas. This supports the Weights of Evidence assumption that there is one training point per unit cell. Sites are first selected to be within the current mask, then selected by the selected methods.&lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;If both site reduction methods are selected Thinning reduction is done first and followed by Random reduction to further reduce the number of sites selected. &lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN /&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;Note! Mask must be of featureclass or featurelayer type. Raster masks are not supported!&lt;/SPAN&gt;&lt;/P&gt;&lt;/DIV&gt;&lt;/DIV&gt;&lt;/DIV&gt;</summary></tool><dataIdInfo><idCitation><resTitle>Training sites reduction</resTitle></idCitation><idAbs>&lt;DIV STYLE="text-align:Left;"&gt;&lt;DIV&gt;&lt;DIV&gt;&lt;P&gt;&lt;SPAN&gt;Reduce number of training sites by random subset of sites or by thinning to one cell per unit areas. This supports the Weights of Evidence assumption that there is one training point per unit cell. Sites are first selected to be within the current mask, then selected by the selected methods.&lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;If both site reduction methods are selected Thinning reduction is done first and followed by Random reduction to further reduce the number of sites selected. &lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN /&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;Note! Mask must be of featureclass or featurelayer type. Raster masks are not supported!&lt;/SPAN&gt;&lt;/P&gt;&lt;/DIV&gt;&lt;/DIV&gt;&lt;/DIV&gt;</idAbs></dataIdInfo><distInfo><distributor><distorFormat><formatName>ArcToolbox Tool</formatName></distorFormat></distributor></distInfo><mdHrLv><ScopeCd value="005"/></mdHrLv><mdDateSt Sync="TRUE">20250205</mdDateSt></metadata>
```

Comment:
Technically the decimal separator depends on the locality settings of the user, so e.g. for me it's a comma (although both work, a period just gets converted to a comma). But I think it's safe to go with the "." instruction since it should work for everyone, even though it might look like it's causing an error.

I realize I never checked the metadata when I did the migration for this tool 😅 The line "Note! Mask must be of featureclass or featurelayer type. Raster masks are not supported!" is no longer necessary, both mask types are supported now. If you want to clean that up in this PR, that's much appreciated 🙂